trivy-fs-scan/action.yaml

```diff
@@ -72,6 +72,37 @@ runs:
       run: |
         touch .trivyignore
 
+    - name: Setup oras
+      uses: oras-project/setup-oras@v1
+
+    - name: Get current date
+      shell: sh
+      id: date
+      run: |
+        echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
```

Comment:
If we're putting the date into the cache, what's the point? It's just going to accumulate cache entries. I would suggest we make caching optional.

Reply:
removed caching . since vulnerabilities are updated daily, we cannot cache it after a day.

Reply:
How does pulling this manually via oras help? the rate limit is on ghcr. In the linked thread, they suggest you can just use their public ECR instead which can be passed as a flag.

Reply:
> TRIVY_DB_REPOSITORY=ghcr.io/aquasecurity/trivy-db,public.ecr.aws/aquasecurity/trivy-db

Seems like this should fallback to the public ECR (should be a similar one for the java db). There's also rate limiting there but we can either set up a user or just hope that having two options should be good enough.